lib/installed-packages-panel.coffee

```diff
@@ -1,6 +1,6 @@
 _ = require 'underscore-plus'
 {$$, TextEditorView, View} = require 'atom-space-pen-views'
-{Subscriber} = require 'emissary'
+{CompositeDisposable} = require 'event-kit'
```

Comment:
Would you mind requiring this from `'atom'` instead and remove the explicit `event-kit` dependency?